phoenix-core/src/main/java/org/apache/phoenix/query/QueryServices.java

```diff
@@ -56,6 +56,9 @@
             "phoenix.query.server.orderBy.spooling.enabled";
     public static final String HBASE_CLIENT_KEYTAB = "hbase.myclient.keytab";
     public static final String HBASE_CLIENT_PRINCIPAL = "hbase.myclient.principal";
+
+    public static final String HBASE_MASTERS = "hbase.masters";
```

Comment:
Is this defined already in a publicly facing hbase module?  If so can we directly include it via hbase compat or similar?